packages/noya-state/src/selectors/themeSelectors.ts

```diff
@@ -62,3 +62,25 @@ export function groupThemeComponents<
 
   return array;
 }
+
+export const getSymbols = (state: ApplicationState): Sketch.SymbolMaster[] => {
+  const symbols: Sketch.SymbolMaster[] = [];
+
+  state.sketch.pages.forEach((p) =>
```

Comment:
We should be able to use `flatMap` here instead of `forEach` and `push`, e.g.

```ts
const symbols = draft.sketch.pages.flatMap(p.layers.flatMap(...))
```

We can also use `Layers.isSymbolMaster(l)` instead of `._class === 'symbolMaster`